tests/viscoelastic_numerical_elastic_time_step.prm

```diff
@@ -0,0 +1,132 @@
+#  Global parameters
```

Comment:
Can you add a description here too?
Something like, this test is exactly like the test XXX, with the only difference being...

I also wondered: The end time of the model is exactly the same as your elastic time step, so both tests do exactly the same thing. Or is this what you're testing? Wouldn't it make sense to have a model where the elastic time step is different from the real time step, and then see if the stress averaging option works and accounts for the differences?

Reply:
I also think it would make sense to add the statistics files to the test outputs, because they include much more information (and digits), so it's much easier to see when things have changed and something is different than before.